### ISSUE TITLE
Breakout rooms

### ISSUE THREAD

**Issue by ruedigerkupper** [nextcloud/spreed#3487]
**Describe the solution you'd like**
Zoom has the concept of »breakout rooms«, allowing the moderator to send participants to separate sub chats in small groups. Groups can be configured on the fly during video calls. Zoom suggests sub groups which the moderator can use and/or reconfigure to her liking.

**Describe alternatives you've considered**
The only alternative that comes to my mind is preparing several conversations in advance and asking participants at some point in the conversation to join "their room– " and return at a given time. This lacks the interactive feature of »breakout rooms« and may confuse participants.

**Comment by wiswedel** [comment]
For inspiration, here's how that's also implemented in BigBlueButton: https://www.youtube.com/watch?v=iR3v-i7R5t0

**Comment by mfstieb** [comment]
this would be great if implemented

**Comment by vagner-dias** [comment]
+1

**Comment by genma** [comment]
In the Nextcloud Special Event announcing "Nextcloud Hub 4" https://www.youtube.com/watch?v=docYiLbGSxI
Breakout rooms are a new features of Talk in this version !

**Comment by nickvergessen** [comment]
Yes, implemented with https://github.com/nextcloud/spreed/issues/8337

**Comment by ruedigerkupper** [comment]
Superb!
